### PR TITLE
test: Add tests for notification helper functions

### LINKED ISSUE #6828: Write tests for notification functions
**Description**
This is a sub issue of #5320
 
Tests are required for notification functions to ensure that they work properly. This increases the test coverage for the code.

**Additional context**
Working on it.

### PR DESCRIPTION
**Pull request by diru1100**
Fixes #6828 
Part of #5320

#### Short description of what this resolves:

Increases test coverage by adding tests for notification helpers functions

#### Changes proposed in this pull request:

- Added  tests for notification helpers functions

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [ ] My branch is up-to-date with the Upstream `development` branch.
- [ ] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

File: tests/all/unit/api/helpers/test_notification.py [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    send_followup_notif_monthly_fee_payment,
+    send_notif_after_event,
+    send_notif_after_export,
+    send_notif_after_import,
+    send_notif_event_role,
+    send_notif_monthly_fee_payment,
+    send_notif_session_accept_reject,
+    send_notif_ticket_purchase_organizer,
+)
+from app.api.helpers.db import save_to_db
```
Comment (niranjan94) [inline tests/all/unit/api/helpers/test_notification.py]: Codacy Issue found: [Unused save_to_db imported from app.api.helpers.db](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5043170)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- tests/all/unit/api/helpers/test_notification.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5043170&bid=16608993)

File: tests/all/unit/api/helpers/test_notification.py [hunk 2]
```diff
@@ -0,0 +1,148 @@
+import unittest
+from app.models import db
```
Comment (niranjan94) [inline tests/all/unit/api/helpers/test_notification.py]: Codacy Issue found: [Unused db imported from app.models](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5043170)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- tests/all/unit/api/helpers/test_notification.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5043170&bid=16608993)

File: tests/all/unit/api/helpers/test_notification.py [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+class TestNotificationHelper(OpenEventTestCase):
+    def test_send_notif_new_session_organizer(self):
+        """Method to test new session notification"""
+        with self.app.test_request_context():
+            user = UserFactory()
+            test_session = SessionFactory()
+            link = common.url_
+            event_name = common.string_
+            current_app.config['TESTING'] = False
```
Comment (iamareebjamal) [inline tests/all/unit/api/helpers/test_notification.py]: Why?

File: tests/all/unit/api/helpers/test_notification.py [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+            invoice_id = 1
+            order_url = common.url_
+            event_name = common.string_
+            subject_id = 1
+            current_app.config['TESTING'] = False
+            send_notif_ticket_purchase_organizer(
+                user, invoice_id, order_url, event_name, subject_id
+            )
+            notif_ = Notification.query.all()
+            self.assertIsNotNone(notif_[0])
```
Comment (iamareebjamal) [inline tests/all/unit/api/helpers/test_notification.py]: All tests should test the actual content of the notification as well

File: tests/all/unit/api/helpers/test_notification.py (hunk 3 above)
Comment (diru1100) [reply tests/all/unit/api/helpers/test_notification.py]: the send_notification isn't storing in db when app config is in TESTING mode

Comment (iamareebjamal) [reply tests/all/unit/api/helpers/test_notification.py]: Then find the reason

**Comment by iamareebjamal** [comment]
@diru1100 Update?

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- tests/all/unit/api/helpers/test_notification.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5043170&bid=16608993)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- tests/all/integration/api/helpers/test_notification.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5043170&bid=16608993)

**Pull request merged**